src/transaction.c

```diff
@@ -113,7 +113,9 @@ char* bigchain_build_json_inputs(BIGCHAIN_INPUT *inputs, uint8_t num_inputs, cha
     }
 
     if(inputs[i].fulfills[0] != '\0') {
-      p = json_str(p, "fulfills", inputs[i].fulfills);
```

Comment:
'fulfills' is an extra json object for TRANSFER and not a string, this had to be fixed.